packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -59,6 +59,7 @@ export class FieldSchema<out Kind extends FieldKind = FieldKind, out Types exten
     get allowedTypeSet(): ReadonlySet<TreeNodeSchema>;
     readonly kind: Kind;
     readonly props?: FieldProps | undefined;
+    readonly requiresValue: boolean;
```

Comment:
For reviewers: this doesn't classify as breaking under our rules since this class is `@sealed`.